Userland/Libraries/LibCrypto/Curves/X25519.cpp

```diff
@@ -0,0 +1,324 @@
+/*
+ * Copyright (c) 2022, stelar7 <dudedbz@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibCrypto/Curves/X25519.h>
+
+namespace Crypto {
+namespace Curves {
```

Comment:
Pls use `namespace Crypto::Curves`, we're in c++20 land now

Reply:
Fixed